packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -256,6 +261,7 @@ export function createReactExtension<
       };
 
       attachComponentData(Result, 'core.plugin', plugin);
+      attachComponentData(Result, 'name', name);
```

Comment:
I don't see the general need for this line - is this the workaround for having an `name` on a widget as an identifier? :)